### DISCUSSION TITLE
Brownie Fund Me - list index out of range

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#57] [Q&A] [answered]
Trying to deploy to ganache-local at getting this error:

Running 'scripts/deploy.py::main'...
  File "brownie/_cli/run.py", line 49, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/deploy.py", line 28, in main
    deploy_fund_me()
  File "./scripts/deploy.py", line 10, in deploy_fund_me
    account = get_account()
  File "./scripts/helpful_scripts.py", line 12, in get_account
    return accounts[0]
  File "brownie/network/account.py", line 113, in __getitem__
    return self._accounts[key]
IndexError: list index out of range

![image](https://user-images.githubusercontent.com/26919212/133926714-d3a3c7a2-f7e1-4c24-b94f-ab88de0e57ea.png)

**Comment by AlbertoSinigaglia** [comment, marked as answer]
please provide the code and the configuration for `ganache-local`

**Reply by toddabraham** [reply]
It's working now.  I was super tired trying to fix this, and so unfortunately I don't know how I fixed it.  Anyhow, onward and upward!

**Comment by sl00k** [comment]
I am having the same error now :(
How do I see the configuration of ganache-local?

**Reply by cromewar** [reply]
> I am having the same error now :(
> How do I see the configuration of ganache-local?

Your get account script is most likely returning None, add ganache local to local blockchains on helpful scripts.

**Reply by sl00k** [reply]
it already was added

**Reply by cromewar** [reply]
Did you added accounts=10 when creating ganache local to brownie ?

**Reply by sl00k** [reply]
Was that a step in the beginning of the video?

This is my config, it has no ganach-local? Should it has one?

![image](https://user-images.githubusercontent.com/33859801/142669380-5462cfc3-4167-4577-9093-716213d4f152.png)

**Reply by cromewar** [reply]
It seems you have ganache-local ok. Could you please paste here your helpful scripts and brownie-config?

**Reply by sl00k** [reply]
![image](https://user-images.githubusercontent.com/33859801/142670275-be05ffe7-3759-4df6-bc1a-cdf4d46ab9c0.png)
![image](https://user-images.githubusercontent.com/33859801/142670309-c6a7c920-e389-4c0f-afde-6325995a88db.png)

**Reply by sl00k** [reply]
![image](https://user-images.githubusercontent.com/33859801/142670628-2367b2a5-ff79-4dbb-87de-0f42b544883c.png)

**Reply by sl00k** [reply]
The code is the same like in the github. Testing on Alchemy worked. So somehow I messed up my local ganache

**Reply by cromewar** [reply]
Huum may be removing it and adding again may help.

**Reply by sl00k** [reply]
sorry I am stuck here.
So many different chains and forks, I got confused.

Local Ganach is my UI-Ganach, right? How do I remove it? Just uninstall and install it again on Windows? Deleting and Adding the network to the list again? And thats it right?

**Reply by cromewar** [reply]
Well is a tricky question the network name is your choose, based on the course is indeed the ganache-ui... by the way you should change the host on ganache ui to wsl in order to work, the http address will change.

![image](https://user-images.githubusercontent.com/20199180/142676404-9d1bd7bf-1a99-4b17-8b0e-9230bf66c078.png)

click on the config button and go to server and change the hostname to wsl

**Reply by sl00k** [reply]
That was the problem. I was not connected to the wsl ... switched in the middle of the video from windows to linux

**Reply by cromewar** [reply]
@sl00k just don't forget to change the host again before closing ganache ui, otherwise the next time it wont startup properly.
If you encounter this error and the host name appears empty you should go to `%APPDATA%` and delete the ganache folder.

**Reply by sl00k** [reply]
thanks for the hint. that would have been the next error :D
